src/SFA.DAS.EmployerIncentives.Web.SystemAcceptanceTests/Steps/Application/ReadyToEnterBankDetailsSteps.cs

```diff
@@ -0,0 +1,150 @@
+﻿using AngleSharp.Html.Parser;
+using FluentAssertions;
+using SFA.DAS.EmployerIncentives.Web.ViewModels.Apply;
+using SFA.DAS.HashingService;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+
+namespace SFA.DAS.EmployerIncentives.Web.SystemAcceptanceTests.Steps.Application
+{
+    [Binding]
+    public class ReadyToEnterBankDetailsSteps : StepsBase
```

Comment:
Please add a feature scope here e.g.

[Scope(Feature = "ReadyToEnterBankDetails")]

this will scope all the method names to just this feature.

Reply:
Done